doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -350,6 +350,11 @@ Other samples
 
 |no_changes_yet_note|
 
+* Added:
+
```

Comment:
Please remove this entry from here and add a comment on PR https://github.com/nrfconnect/sdk-nrf/pull/25910 after this PR is merged as this PR seems to be for the 3.2 release. The changelog file is cleaned for the next release.